src/main/java/net/minecraftforge/event/world/DimensionPreloadEvent.java

```diff
@@ -0,0 +1,35 @@
+package net.minecraftforge.event.world;
+
+import net.minecraft.world.DimensionType;
+import net.minecraftforge.common.DimensionManager;
+import net.minecraftforge.fml.common.eventhandler.Cancelable;
+import net.minecraftforge.fml.common.eventhandler.Event;
+import net.minecraftforge.common.MinecraftForge;
+
+/**
+ * Created by goreacraft on 17/03/03.
```

Comment:
can you help me explain what would be the proper way of fixing this now? 
should i add a new commit and remove this one? or there is a way to edit this things on the fly